joint_limits_controller/src/joint_limiter.cpp

```diff
@@ -0,0 +1,85 @@
+#include <joint_limits_controller/limited_joint_handle.h>
+
+double JointLimiter::limitBounds(double value, double min, double max){
+    if(value > max) return max;
+    else if (value < min) return min;
+    return value;
+}
+
+std::pair<double,double> JointLimiter::getSoftBounds(double value, double k, double lower, double upper){
+    return std::make_pair(-k*(value-lower), -k*(value-upper));
```

Comment:
is absolute value needed here for `(value-lower)` and `(value -upper)` respectively?